src/solver/utils/ortools_utils.cpp

```diff
@@ -331,8 +349,8 @@ const std::map<std::string, struct OrtoolsUtils::SolverNames> OrtoolsUtils::mpSo
 
 const std::map<std::string, math_opt::SolverType> OrtoolsUtils::mathoptSolverMap = {
   {"pdlp", math_opt::SolverType::kPdlp},
-  {"scip", math_opt::SolverType::kGscip},
-  {"xpress", math_opt::SolverType::kXpress}};
+  {"scip", math_opt::SolverType::kGscip}/*,
+  {"xpress", math_opt::SolverType::kXpress}*/};
```

Comment:
Why is this commented ?

Reply:
I forgot to remove this comment. Now fixed.